locale/es_MX/LC_MESSAGES/clr-installer.po

```diff
@@ -398,11 +398,11 @@ msgstr "MX: Confirm"
 msgid "You can use alpha-numeric characters and hyphens with a maximum of %d characters"
 msgstr "MX: You can use alpha-numeric characters and hyphens with a maximum of %d characters"
 
-msgid "Welcome to Clear Linux* Desktop Installation"
+msgid "Welcome to Clear Linux* OS Desktop Installation"
 msgstr "Bienvenido a la instalación de Clear Linux* para computadoras de escritorio"
 
 #, c-format
-msgid "Thank you for choosing Clear Linux*"
+msgid "Thank you for choosing Clear Linux* OS"
 msgstr Gracias por elegir Clear Linux*"
```

Comment:
I know that the final translation will fix this, but for the time being can you please modify the translation to:

msgstr "Gracias por elegir **operativo** Clear Linux*"